[TST] Upcoming dependency test failures

The weekly build with nightly wheels from numpy and pandas
has failed. Check the logs for any updates that need to be
made in matplotlib.

Only a single failure:
```
=================================== FAILURES ===================================
________________________ TestAsinhLocator.test_fallback ________________________
[gw0] linux -- Python 3.8.12 /opt/hostedtoolcache/Python/3.8.12/x64/bin/python

self = <matplotlib.tests.test_ticker.TestAsinhLocator object at 0x7ff30e27a3a0>

    def test_fallback(self):
        lctr = mticker.AsinhLocator(1.0, numticks=11)
    
>       assert_almost_equal(lctr.tick_values(100, 101),
                            np.arange(100, 101.01, 0.1))
E       AssertionError: 
E       Arrays are not almost equal to 7 decimals
E       
E       (shapes (2,), (11,) mismatch)
E        x: array([ 10., 100.])
E        y: array([100. , 100.1, 100.2, 100.3, 100.4, 100.5, 100.6, 100.7, 100.8,
E              100.9, 101. ])

lib/matplotlib/tests/test_ticker.py:513: AssertionError
```

--- a/lib/matplotlib/tests/test_ticker.py
+++ b/lib/matplotlib/tests/test_ticker.py
@@ -510,8 +510,8 @@ class TestAsinhLocator:
     def test_fallback(self):
         lctr = mticker.AsinhLocator(1.0, numticks=11)
 
-        assert_almost_equal(lctr.tick_values(100, 101),
-                            np.arange(100, 101.01, 0.1))
+        assert_almost_equal(lctr.tick_values(101, 102),
+                            np.arange(101, 102.01, 0.1))
 
     def test_symmetrizing(self):
         class DummyAxis:

